### DISCUSSION TITLE
Adjust Configuration with MCU Path

### DISCUSSION THREAD

**Discussion by paulleywalnuts** [bassamanator/Sovol-SV06-firmware#112] [Q&A] [answered]
I am trying to install Klipper on my SV06Plus with a raspberry pi 4b, however, I am having issues with this step. When I run the by-id or by-path command, it does not return the expected results and says it cannot be found.

**Comment by bassamanator** [comment, marked as answer]
https://talky.io/two-pieces-of-cornbread-and-one-cornbread
